Modules/posixmodule.c

```diff
@@ -5309,52 +5311,68 @@ os_spawnve_impl(PyObject *module, int mode, path_t *path, PyObject *argv,
 
 
 #ifdef HAVE_FORK
+
+/* Helper function to validate arguments.
+   Returns 0 on success.  1 on failure with a TypeError raised.
```

Comment:
It is more common to return either
* 1 on success, 0 on failure, or
* 0 on success, -1 on failure.